data/README.txt

```diff
@@ -0,0 +1,58 @@
+Running the HAFS global requires GFS physics (sfc) and dynamics (atm)
+files at the initialization time. The atm_ic job will interpolate this
+to the grid of interest.
+
+Unfortunately, the HAFS workflow lacks a job to pull the data it needs
+from HPSS. Typically, a role account (emc.hurdata) owns the data
+everyone needs. That data is maintained by one or two people manually.
+
+This directory has the scripts they use. These scripts are not
+elegant, polished, workflow scripts intended for general
+consumption. They're personal scripts used by a few HAFS developers.
+
+To use them...
+
+1. Pick a directory in a location with lots of disk space.
+
+    mkdir -p /path/to/big/place
+
+2. Copy the three scripts into that location. Be certain to preserve execution flags (cp -p)
+
+    cp -fp /path/to/HAFS/data/*.sh /path/to/big/place/.
+
+3. cd to that location
+
+    cd /path/to/big/place
+
+4. Edit the #SBATCH --acount lines at the top of these scripts. Change it to a project you can use.
```

Comment:
For dates prior to 2021032200, the gfsv16retro script is needed.  
Otherwise, only the gfsv16prod script is used.

The driver script also specifies an account that must be edited.  Search for the line starting with BSUB in the code block that is appropriate for the machine being used.  Edit the --account in this line.

Reply:
The BSUB lines aren't used. That's for the LSF scheduler, which no NOAA machine use anymore.

Reply:
Please ignore my prior comment. I misread what you said.

Reply:
I've updated it so you don't need to change the scripts. Instead, the account is the first argument to the driver script. The README.txt has the new instructions.